recipes/orange3/meta.yaml

```diff
@@ -0,0 +1,76 @@
+{% set version = "3.3.6" %}
+
+package:
+  name: orange3
+  version: {{ version }}
+
+source:
+  fn: Orange3-{{ version }}.tar.gz
+  url: https://github.com/biolab/orange3/archive/{{ version }}.tar.gz
+  md5: 927f3702ea5c9f1c8d1b59478476b4d5
+
+build:
+  number: 0
+  script:
+    # FIXME: This is a hack to make sure the environment is activated.
+    # The reason this is required is due to the conda-build issue
+    # mentioned below.
+    #
+    # https://github.com/conda/conda-build/issues/910
+    #
+    - source activate "${CONDA_DEFAULT_ENV}"  # [unix]
```

Comment:
Should I update the recipe in some way? I am not sure why the FIXME had to be added as the build succeeded without it.